Apps/WebClient/src/ClientApp/src/main.ts

```diff
@@ -88,6 +92,9 @@ store.dispatch("config/initialize").then((config: ExternalConfiguration) => {
     );
 
     // Initialize services
+    loggingService.initialize();
+    loggingService.info("Winston Logger initialized.");
```

Comment:
Let's not put the implementation name directly here since it can change. If logging is required for its initialization lets move it to the implementation initializer

Reply:
Oh this is just a normal info log, as we code, we will have to write a lot of these at the info or debug level. The idea is when code gets pushed to PROD and if we want to log more at info or debug level, we can turn these on for more info, debug, verbose for debugging.

Reply:
It's fine it logs, just not here since at this point the code is agnostic of the implementation being used. The log should be placed on the initialize() step of the logger

Reply:
Interesting approach :-) Mine is to log after calling a method. Let's chat to make sure

Reply:
btw, I'm good to remove this for now as I'm about to introduce a "functional way" to log these in the next tasks :-)